### PR TITLE
cpp: fixup use of Status::OK

### PR DESCRIPTION
**Pull request by daviddrysdale**

### CODE REVIEW

**Comment by daviddrysdale** [comment]
Recreated pull #1371 after I messed up branch locations.

File: cpp/util/status_test_util.h [hunk 1]
```diff
@@ -83,9 +83,9 @@ ::util::Status ToStatus(const ::util::StatusOr<T>& statusor) {
 
 
 #define EXPECT_OK(expr) \
-  EXPECT_EQ(util::Status::OK, util::testing::ToStatus(expr))
+  EXPECT_EQ(::util::OkStatus(), util::testing::ToStatus(expr))
 #define ASSERT_OK(expr) \
-  ASSERT_EQ(util::Status::OK, util::testing::ToStatus(expr))
+  ASSERT_EQ(::util::OkStatus(), util::testing::ToStatus(expr))
```
Comment (RJPercival) [inline cpp/util/status_test_util.h]: Still some more places where `ASSERT_OK` could be used.

Comment (daviddrysdale) [inline cpp/util/status_test_util.h]: Er, not for this one :-)

(look at the line above)

Comment (RJPercival) [inline cpp/util/status_test_util.h]: Oops! LGTM.

**Comment by daviddrysdale** [comment]
Travis seems borked atm so I'll try restarting builds tomorrow.

**Pull request merged**